packaging/ubuntu-14.04/rules

```diff
@@ -145,7 +145,8 @@ override_dh_auto_build:
 	$(MAKE) -C data all
 
 override_dh_auto_test:
-	dh_auto_test -- -mod=vendor $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS) $(DH_GOPKG)/...
+	GO111MODULE=on \
```

Comment:
One silly question - could we simply set GO111MODULE=on globally ? or would that break in some other place then?

Reply:
Let's try that in a followup. I would like to have the LP builds unblocked first.